### DISCUSSION TITLE
Developing Grove Vision AI V2 using Himax SDK | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1627] [Q&A]
# Developing Grove Vision AI V2 using Himax SDK | Seeed Studio Wiki

Introduces how to use Himax's SDK to develop Grove Vision AI V2.

https://wiki.seeedstudio.com/grove_vision_ai_v2_himax_sdk/

**Comment by benclockworks** [comment]
this turtorial doesn't work with mac OS :/arm-gnu-toolchain-13.2.Rel1-x86_64-arm-none-eabi/bin/arm-none-eabi-gcc: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 2.6.32, BuildID[sha1]=b748bab9979bca0be95826197975d808e1a3d725, not stripped

**Reply by benclockworks** [reply]
for mac users just use https://developer.arm.com/downloads/-/gnu-rm mac (notarized ) package and keep running make over and allow all the dependencies access over and over

**Comment by zano17** [comment]
I wonder  if  it is  possible  to use  ubuntu  22  directly  without  windows   ie  installing  ubuntu  22  on  wsl  of  windows ?

**Reply by nfs0619** [reply]
Yes, you can use Ubuntu 22 directly without Windows. In that case, you can continue to the [Preparation of the compilation environment](https://wiki.seeedstudio.com/grove_vision_ai_v2_himax_sdk/#preparation-of-the-compilation-environment).
In our documentation we used a Windows PC as an example, so WSL was needed to provide a Linux environment for building the Himax SDK.
